controllers/guacamole.go

```diff
@@ -46,7 +46,7 @@ var UpGrader = websocket.Upgrader{
 	Subprotocols: []string{"guacamole"},
 }
 
-func (c *ApiController) GetAssetTunnel() error {
+func (c *ApiController) Guacamole() error {
```

Comment:
Guacamole is unclear and not a good API name. What's wrong with the original name `GetAssetTunnel`?